docs/user/integration/dynatrace/README.md

```diff
@@ -259,3 +262,30 @@ For easier access from the Kyma dashboard, add links to new navigation under **D
     ```
 
 2. If your Secret has a different name or namespace, then download the file first and adjust the namespace and name accordingly in the 'dataSources' section of the file.
+
+## Use Dynatrace Dashboards
+
+1. To view metrics in Dynatrace dashboards, import the file [Telemetry Module Status](./telemetry-resource-metrics.json) as a Dynatrace dashboard. For details, see [Importing Dashboards](https://docs.dynatrace.com/docs/analyze-explore-automate/dashboards-classic/dashboards/dashboard-json#import-dashboard).
```

Comment:
We are talking here about a specific dashboard, nothing generic, so I would start with:

To visualize the health of the Kyma Telemetry module and it's related pipelines, import manually the file ...

Reply:
Suggestion with language editing - and I argue we can drop the "manually" - the linked instructions don't offer any automatic alternative?

```suggestion
1. To see the health of the Kyma Telemetry module and its related pipelines, import the file [Telemetry Module Status](./telemetry-resource-metrics.json) as a Dynatrace dashboard. For details, see [Importing Dashboards](https://docs.dynatrace.com/docs/analyze-explore-automate/dashboards-classic/dashboards/dashboard-json#import-dashboard).
```

Reply:
Sounds awesome, let's do it